data/Templates/FhirToHL7v2/DataType/_CX-ExtendedID.liquid

```diff
@@ -7,7 +7,13 @@ id = {
         "scheme": "",
     },
     "assigning": {
-        "authority": "",
+        "authority": {
+            "namespace": "",
+            "universal": {
```

Comment:
Is this real FHIR? I only ask because I had to also implement HD data type for PID.3 and my implementation looks way different. I didn't see a structure like this and the way that the HL7 -> FHIR conversion works seems to append the universal.type to a URL and completely discard the universal.id. For instance, for an identifier the structure that should be fed into CX.4 is this:

```
"assigner": {
   "reference": "Organization/4e68b44e-c053-02b1-6d6d-fe816f9fdfec"
},
```

Reply:
I think we sorted this offline but let me know if you're still confused about this.